v2/codetags/types.go

```diff
@@ -17,20 +17,32 @@ limitations under the License.
 package codetags
 
 import (
-	"fmt"
+	"strconv"
 	"strings"
 )
 
 // TypedTag represents a single comment tag with typed args.
 type TypedTag struct {
```

Comment:
For consideration: does "Typed" add much here?  Should this be Tag or should Arg be TypedArg ?

Reply:
After moving to a dedicated package, `Tag` is sufficient.  There was another Tag type in root package that this started in.

Fixed.